app/Controller/PagesController.php

```diff
@@ -74,3 +120,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
```

Comment:
__[staging][code_sniffer]__
* You must use "/**" style comments for a class comment
* Opening brace of a class must be on the line after the definition